xmipp

```diff
@@ -379,11 +379,10 @@ def getScipionHome():
 
 
 def compileModule(Nproc,module):
-    shutil.copyfile(Config.FILE_NAME,"src/%s/install/%s" % (module, Config.FILE_NAME))
     if module == "xmipp":
         stampVersion()
     log = []
-    ok = runJob("scons -j%s" % Nproc, "src/%s" % module, log=log)
+    ok = runJob("/usr/bin/env python3 $(which scons) -j%s" % Nproc, "src/%s" % module, log=log)
```

Comment:
works with python 2.7 out of scipion3 env?

Reply:
No. Python3 is required (also for scipion, so it should be there). This change makes sure that scons is using python3 too, even though python2 might be set as default on the machine.

Reply:
Ok, thakns